plugins/net_plugin/net_plugin.cpp

```diff
@@ -1707,8 +1705,8 @@ namespace eosio {
 
       uint32_t lib_num = 0;
       uint32_t fork_head_block_num = 0;
-      std::tie( lib_num, std::ignore, fork_head_block_num,
-                std::ignore, std::ignore, std::ignore ) = my_impl->get_chain_info();
+      std::tie( lib_num, fork_head_block_num,
+                std::ignore, std::ignore ) = my_impl->get_chain_info();
 
```

Comment:
I think it would be cleaner if `get_chain_info()` returned a `chain_info_t` struct with 4 members, instead of a tuple and the  extra variable declarations.

So these 4 lines would be replaced with:

`auto chain_info = my_impl->get_chain_info();`

Reply:
Done.

Reply:
Nice, thank you @heifner